s3compatapi/src/main/java/com/snowflake/s3compatapitestsuite/compatapi/S3CompatStorageClient.java

```diff
@@ -98,12 +100,27 @@ public String getBucketLocation(String bucketName) {
                 if (correctRegion != null) {
                     this.s3Client.setSignerRegionOverride(correctRegion);
                     this.s3Client.setRegion(RegionUtils.getRegion(correctRegion));
-                    return this.s3Client.getBucketLocation(bucketName);
+                    return this.s3Client.getBucketLocation(bucketName).toLowerCase();
```

Comment:
why not direct return `correctRegion`? Why additional API call: `this.s3Client.getBucketLocation(bucketName)`?